seed/seed.go

```diff
@@ -163,7 +163,7 @@ func (s *Seeder) Seed(cfg *config.Config) error {
 			// the parsing won't stop, and we make sure we try to parse every flag passed when running the command.
 			for _, arg := range os.Args[1:] {
 				if err := flagSet.Parse([]string{arg}); err != nil {
-					// Simply log errors that can happen, such as parsing unexpected flags. We want this to be silent
+					// Simply log errors that can happen, such as parsing unexpected flags. We want this to be silent,
```

Comment:
Very naive comment, but would it be better to move the comment after the period (`.`) to the next line ?

Reply:
There is no guideline for that. Personally I am optimizing for vertical space and only go to the next line if the content does not fit the width. More can be displayed this way, l;less scrolling.